_extensions/mpim/slides.scss

```
@@ -136,3 +136,8 @@ div.column {
 .reveal .slide-number > a[href*='title-slide'] {
   display: none;
 }
+
+.img-polaroid {
```

Probably this could be also just
```suggestion
.polaroid {
```
? It should also work on other boxes (e.g. `div`s) and might be useful in those cases as well. But I'd also be fine keeping it as it's currently.

---

Yes, that makes sense 👍 Updated accordingly